x/oracle/keeper/keeper.go

```diff
@@ -0,0 +1,108 @@
+package keeper
+
+import (
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+
+	"github.com/provenance-io/provenance/x/oracle/types"
+)
+
+type Keeper struct {
+	cdc        codec.BinaryCodec
+	storeKey   storetypes.StoreKey
+	memKey     storetypes.StoreKey
+	paramstore paramtypes.Subspace
```

Comment:
The `x/params` module is deprecated. It's "replacement" is to have each module do their own thing for params. E.g. in the `x/sanction` module, they're stored as `<type byte> | <field name> -> <field value>`. See the stuff below https://github.com/provenance-io/cosmos-sdk/blob/main-pio/x/sanction/keeper/keeper.go#L287.